pkg/apply/merge.go

```diff
@@ -91,6 +93,17 @@ func mergeServiceForUpdate(current, updated *uns.Unstructured) error {
 			}
 		}
 
+		clusterIPs, found, err := uns.NestedStringSlice(current.Object, "spec", "clusterIPs")
+		if err != nil {
+			return err
+		}
+		if found {
+			err = uns.SetNestedStringSlice(updated.Object, clusterIPs, "spec", "clusterIPs")
+			if err != nil {
+				return err
+			}
+		}
+
 		ipFamily, found, err := uns.NestedString(current.Object, "spec", "ipFamily")
```

Comment:
In master, `ipFamily` no longer exists, but we should be copying `ipFamilies` and `ipFamilyPolicy`. (You'd only notice that being a problem if you were running with the dual-stack feature gate, which is why you didn't notice that.)

Reply:
Good catch, updated.